08-CMSIS/Makefile

```diff
@@ -43,11 +43,11 @@ $($(1)_TARGET):
 endef
 
 #Create corresponding build command to the device
-$(eval $(call eval_build_command,$(STM32P103_DEVICE)))
 $(eval $(call eval_build_command,$(STM32F429_DEVICE)))
+$(eval $(call eval_build_command,$(STM32P103_DEVICE)))
 
 #default target
-PLAT ?= $(STM32P103_DEVICE)
+PLAT ?= $(STM32F429_DEVICE)
```

Comment:
Don't change the default target. Did you verify the changes on qemu-based stm32-p103 as well?